ext/nnstreamer/tensor_filter/tensor_filter_python.cc

```diff
@@ -41,7 +41,16 @@
 #pragma GCC diagnostic ignored "-Wformat"
 #endif
 
+/**
+ * NOTE: This workaround is only for python2.7. If we are going to use external
+ * open-source projects written in cpp with the 'register' keyword, we need to
+ * project-widely apply this.
+ */
+#if __cplusplus > 199711L
```

Comment:
Q: what `199711L` means? As I search, the value of C++11 is `201103L`.
https://gist.github.com/ax3l/53db9fa8a4f4c21ecc5c4100c0d93c94#meaning-of-__cplusplus

Reply:
@helloahn > 199711L which means >= 201103L

Reply:
I got it :)